testapi.pm

```diff
@@ -897,19 +897,24 @@ sub script_sudo {
 
 =head2 script_output
 
-  script_output($script [, $wait])
+  script_output($script [, $wait, type_command => 1])
 
-fetches the script through HTTP into the SUT and execs it with C<bash -xe> and directs
-C<stdout> (I<not> C<stderr>!) to the serial console and returns the output I<if> the script
-exists with 0. Otherwise the test is set to failed.
+Executing script inside SUT with C<bash -eox> (in case of serial console with C<bash -eo>)
+and directs C<stdout> (I<not> C<stderr>!) to the serial console and returns
+the output I<if> the script exists with 0. Otherwise the test is set to failed.
 
-The default timeout for the script is 30 seconds. If you need more, pass a second parameter
+Script content based on C<current_test_script> value and can be fetched  through HTTP
+into the SUT or typed with C<type_string>. This depends on C<current_test_script> length,
```

Comment:
that's a bit technical for a documentation - even for API documentation. I would leave it at 'if the command is typed or fetched through HTTP depends on various parameters, so if you need to have it typed because your SUT is offline, pass type_command => 1'